### PR TITLE
rename functions, destructure props, change keys

### PR DESCRIPTION
**Pull request by ExuLu**

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -7,8 +7,11 @@ import uniqid from 'uniqid';
 
 function App() {
   const [notes, setNotes] = useState([]);
+  const [key, setKey] = useState(1);
+  
   function addNote(inputText) {
-    setNotes([...notes, inputText]);
+    setNotes([...notes, { ...inputText, key }]);
+    setKey((key) => key + 1);
```
Comment (a13xg0) [inline src/components/App.jsx:13]: An excellent choice of using the setter function! I love it!

File: src/components/App.jsx [hunk 2]
```diff
@@ -20,7 +23,7 @@ function App() {
       <CreateArea onClick={addNote} />
       {notes.map((note, index) => (
         <Note
-          key={uniqid()}
```
Comment (a13xg0) [inline src/components/App.jsx:23]: As we stop using the `uniqid` it is a good practice to clean imports and remove this package from the project

**Review by a13xg0 — CHANGES_REQUESTED** [review]
Great job on updating your code with best regarding naming and handling keys in React components. 

Let's cleanup unused libraries and we are good to go! Btw, with this kind of issues eslint is really shines!

**Comment by ExuLu** [comment]
delete unnessary code

**Pull request merged**